### PR TITLE
 AUTH-5428: added app launcher customization fields to access applica…

### PR DESCRIPTION
**Pull request by russellkerns**
## Description

This adds the ability to update the access app launcher application with customized display settings
## Has your change been tested?

build the provider locally and used the fields on my own account

unit tests

## Types of changes

What sort of change does your code introduce/modify?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [] This change is using publicly documented in [cloudflare/api-schemas](https://github.com/cloudflare/api-schemas) 
      and relies on stable APIs.

[1]: https://help.github.com/articles/closing-issues-using-keywords/

### CODE REVIEW

File: access_application.go [hunk 1]
```diff
@@ -116,6 +117,26 @@ type SaasApplication struct {
...
 
+type AppLauncherCustomization struct {
+	LandingPageDesign     LandingPageDesign `json:"landing_page_design"`
+	LogoURL               string            `json:"app_launcher_logo_url"`
+	HeaderBackgroundColor string            `json:"header_bg_color"`
+	BackgroundColor       string            `json:"bg_color"`
+	FooterLinks           []FooterLink      `json:"footer_links"`
+}
+
+type FooterLink struct {
```
Comment (jacobbednarz) [inline access_application.go]: as this isn't namespaced, can we prefix this with `Access` to make it clear what the purpose is?

File: access_application.go [hunk 2]
```diff
@@ -116,6 +117,26 @@ type SaasApplication struct {
...
+	Name string `json:"name"`
+	URL  string `json:"url"`
+}
+
+type LandingPageDesign struct {
```
Comment (jacobbednarz) [inline access_application.go]: as this isn't namespaced, can we prefix this with `Access` to make it clear what the purpose is?

**Comment by jacobbednarz** [comment]
lgtm - let me know when this lands in the API schemas and we can merge this.

**Comment by russellkerns** [comment]
> lgtm - let me know when this lands in the API schemas and we can merge this.

@jacobbednarz this is in the api schemas now! should be good to go

**Pull request merged**